index.js

```diff
@@ -38,6 +50,17 @@
 
 // ***** Question 4 *****
 
+function sameSameButDifferent(num , maybeNum) {
+  if (num === maybeNum) {
+    return "same same"
+  } else if (num == maybeNum) {
```

Comment:
This works, but you can also convert the string to a number with `parseInt` to avoid `==`:

```js
} else if (num === parseInt(maybeNum)) {
```